lib/std/testing.zig

```diff
@@ -501,7 +501,10 @@ fn printIndicatorLine(source: []const u8, indicator_index: usize) void {
         while (i < indicator_index) : (i += 1)
             print(" ", .{});
     }
-    print("^\n", .{});
+    if (indicator_index >= source.len)
+        print("^ (out of bounds)\n", .{})
```

Comment:
nit: perhaps call this `(end of string)`?

Reply:
That's better I think!